src/auro-select.js

```diff
@@ -6,6 +6,8 @@
 // If using litElement base class
 import { LitElement, html } from "lit-element";
 
+import '@aurodesignsystem/auro-menu';
```

Comment:
We can't do this. This breaks the peer dependency rule. Right now in this code `@aurodesignsystem/auro-menu` is listed as a peer and a dev dependency. When we ship the component, `@aurodesignsystem/auro-menu` will not be part of the install in a prod application. 

Also, this means that we need to maintain the version update of `@aurodesignsystem/auro-menu` in this element and release select whenever there is a menu update.

Reply:
This change is coming from a collaboration with @jason-capsule42. This import was added because of a race condition where `auro-menu` would not be loaded completely before the example was trying to run, causing all of the examples in `api.js` to fail initialization.

This was done similarly in combobox a while back, which is why we decided to do that same thing here.

https://github.com/AlaskaAirlines/auro-combobox/blob/main/src/auro-combobox.js

Reply:
Ok, we need to find a better way to manage this. The more we do this, the more additional maintenance we have to mange which creates a larger burden. 

Also, you need to update the package.json. You need to define menu as a dependency.